app/controllers/application_controller.rb

```diff
@@ -1,7 +1,12 @@
+# frozen_string_literal: true
+
 class ApplicationController < ActionController::Base
   layout :layout_by_resource
 
+  # This causes the sign out link to not be displayed -- somehow user_signed_in?
```

Comment:
`This` 

what exactly?

Reply:
Lol. Debugging comment to self I forgot to delete. Thanks :)